frontend/src/metabase/meta/Dashboard.js

```diff
@@ -29,67 +30,81 @@ export type ParameterSection = {
   options: ParameterOption[],
 };
 
-export const PARAMETER_SECTIONS: ParameterSection[] = [
-  {
-    id: "date",
-    name: t`Time`,
-    description: t`Date range, relative date, time of day, etc.`,
-    options: PARAMETER_OPERATOR_TYPES["date"].map(option => {
-      return {
-        ...option,
-        sectionId: "date",
-        combinedName: getOperatorDisplayName(option, "date", t`Date`),
-      };
-    }),
-  },
-  {
-    id: "location",
-    name: t`Location`,
-    description: t`City, State, Country, ZIP code.`,
-    options: PARAMETER_OPERATOR_TYPES["string"].map(option => {
-      return {
-        ...option,
-        sectionId: "location",
-        combinedName: getOperatorDisplayName(option, "string", t`Location`),
-      };
-    }),
-  },
-  {
-    id: "id",
-    name: t`ID`,
-    description: t`User ID, product ID, event ID, etc.`,
-    options: [
-      {
-        ..._.findWhere(PARAMETER_OPTIONS, { type: "id" }),
-        sectionId: "id",
-      },
-    ],
-  },
-  {
-    id: "number",
-    name: t`Number`,
-    description: t`Subtotal, Age, Price, Quantity, etc.`,
-    options: PARAMETER_OPERATOR_TYPES["number"].map(option => {
-      return {
-        ...option,
-        sectionId: "number",
-        combinedName: getOperatorDisplayName(option, "number", t`Number`),
-      };
-    }),
-  },
-  {
-    id: "category",
-    name: t`Other Categories`,
-    description: t`Category, Type, Model, Rating, etc.`,
-    options: PARAMETER_OPERATOR_TYPES["string"].map(option => {
-      return {
-        ...option,
-        sectionId: "category",
-        combinedName: getOperatorDisplayName(option, "string", t`Category`),
-      };
-    }),
-  },
-];
+const areFieldFilterOperatorsEnabled = () =>
+  MetabaseSettings.get("field-filter-operators-enabled?");
+
+export function getParameterSections(): ParameterSection[] {
```

Comment:
`PARAMETER_SECTIONS` (and `PARAMETER_OPTIONS` in `meta/Parameters.js`) now depend on the `field-filter-operators-enabled?` flag, so we need to build this options upon component instantiation now instead of globally on app instantiation since the flag is resolved asynchronously.